### DISCUSSION TITLE
Authentication error:  status code 400

### DISCUSSION THREAD

**Discussion by FL350** [homebridge-simplisafe3/homebridge-simplisafe3#351] [Q&A] [answered]
Hi  - I haven't been able to auth. I've tried both using the UI and manually via terminal but get the same error below:

> [21/07/2022, 17:51:45] Homebridge v1.5.0 (HAP v0.10.2) (Homebridge 2A1C) is running on port 51719.
> [21/07/2022, 17:52:18] [Homebridge UI] [homebridge-simplisafe3] Incoming Request: /credentialsExist
> [21/07/2022, 17:52:18] [Homebridge UI] [homebridge-simplisafe3] Incoming Request: /nLoginSteps
> [21/07/2022, 17:52:53] [Homebridge UI] [homebridge-simplisafe3] Incoming Request: /initiateLoginAndAuth
> [21/07/2022, 17:52:53] [Homebridge UI] [homebridge-simplisafe3] Loading login auth url...
> [21/07/2022, 17:52:54] [Homebridge UI] [homebridge-simplisafe3] Attempting to login with credentials...
> [21/07/2022, 17:52:54] [Homebridge UI] [homebridge-simplisafe3] Authentication error: Request failed with status code 400
> [21/07/2022, 17:52:59] [Homebridge UI] [homebridge-simplisafe3] Terminating child process...
> [21/07/2022, 17:52:59] [Homebridge UI] [homebridge-simplisafe3] Child process ended

I don't see anything in the README or in discussions. For reference, I am using it on Pi running Adguard and the install seems to have gone to /home/homebridge. Not sure what's going on.

**Comment by jwiemer** [comment]
Same! I'm trying to set up the verified Simplisafe plugin on Mac OS (11.6.5) and I keep getting a 400 authentication error. I can successfully login to Simplisafe's website with the same credentials. The documentation doesn't explain what to do in this situation. Thanks in advance for the help!
![image](https://user-images.githubusercontent.com/6116557/180304759-576ba351-7855-451a-b87a-0366a4714589.png)
![image](https://user-images.githubusercontent.com/6116557/180304795-34330bee-39f8-4c8b-9ff4-9c25118d252a.png)

**Comment by shamoon** [comment, marked as answer]
Unfortunately it looks like SS added reCAPTCHA to the login which is likely what’s breaking this. Sadly that probably means we have to go back to logging in via the browser. I expect this will require a new release of course but I will have to find some time to do that, no clear ETA yet as I’m a bit short on time at the moment. Will track this in #352

**Reply by FL350** [reply]
Thanks, @shamoon. That is disappointing. Hoping you find some time to fix it but totally understandable it might take time. Thanks for what you do.

**Reply by shamoon** [reply]
Hey guys, I rereleased v1.10.7 which resolves this by going back to the old method of using the browser, sadly. Feel free to nag SS about it (be polite =) but I dont expect they will help and this is not likely something we will be able to circumvent. The old method does still work of course.

**Comment by FL350** [comment]
Worked like a charm, @shamoon. Thanks so much! I’m amazed how awesome this works.
